packages/transport-native/src/index.tsx

```diff
@@ -0,0 +1,41 @@
+import { NativeModules, Platform } from 'react-native';
+
+const LINKING_ERROR =
+    `The package 'trezor-transport-native' doesn't seem to be linked. Make sure: \n\n${Platform.select(
+        { ios: "- You have run 'pod install'\n", default: '' },
+    )}- You rebuilt the app after installing the package\n` +
+    `- You are not using Expo managed workflow\n`;
+
+console.log(NativeModules);
+
+const TrezorTransport = NativeModules.TrezorTransport
```

Comment:
good use case for [Nullish Coalescing](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#nullish-coalescing) 

Instead of `A ? A : B` you could type `A ?? B`.

Reply:
I think this is not good place for it, in case it won't be linked it will cause errors like `calling ememurate of undefined` etc. This solution with get proxy will produce nice error message where you will exactly know that something went wrong during package linking

Reply:
Never mind, I got what you wanted me to do... that is actually good idea I will use it

Reply:
great idea, used